src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala

```diff
@@ -29,7 +29,7 @@ object SbtHeader extends AutoPlugin {
   object autoImport {
 
     object HeaderPattern {
-      val cStyleBlockComment = """(?s)(/\*(?!\*).*?\*/(?:\n|\r|\r\n)+)(.*)""".r
```

Comment:
I mean your definition of a copyright header is just the first comment in the file anyway.